src/components/AgenciesPage/AgenciesPage.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import { Container, Grid } from '@mui/material';
+
 import { AgenciesList } from './components';
 import './AgenciesPage.scss';
 
 export const AgenciesPage = () => (
-  <section className="agencies-page">
+  <Container maxWidth="xl">
     <h1 className="agencies-header">Agencies</h1>
     <AgenciesList />
-  </section>
```

Comment:
Why did you remove the section tag here?

Reply:
You can delete `section` if you use `Container`

Reply:
deleted